lucene/core/src/java/org/apache/lucene/codecs/lucene90/Lucene90VectorReader.java

```diff
@@ -120,6 +120,7 @@ private static IndexInput openDataInput(
 
     String fileName =
         IndexFileNames.segmentFileName(state.segmentInfo.name, state.segmentSuffix, fileExtension);
+    // nocommit should the openInput call be put in try-finally as well for IOException handling?
```

Comment:
I think it's OK - the exception handling got moved to the outer scope in the PR I referenced above

Reply:
Sounds good. I have removed the nocommit.